siriuspy/siriuspy/clientconfigdb/types/as_diagnostics.py

```diff
@@ -620,7 +620,7 @@ def get_dict():
     [':INFOTBTRate-SP', 382, 0.0],
```

Comment:
Do you think it would make sense to take the opportunity to change all occurrences of "TBT" to "TbT"? We seem to have a mix of it in PV names and enum descriptions.

Reply:
Very good idea @ericonr! Standardization rocks!

Reply:
done!